oap-server/server-configuration/configuration-zookeeper/src/main/java/org/apache/skywalking/oap/server/configuration/zookeeper/ZookeeperConfigWatcherRegister.java

```diff
@@ -46,9 +51,28 @@ public ZookeeperConfigWatcherRegister(ZookeeperServerSettings settings) throws E
     @Override
     public Optional<ConfigTable> readConfig(Set<String> keys) {
         ConfigTable table = new ConfigTable();
-        keys.forEach(s -> {
-            ChildData data = this.childrenCache.getCurrentData(this.prefix + s);
-            table.add(new ConfigTable.ConfigItem(s, data == null ? null : new String(data.getData())));
+        keys.forEach(key -> {
+            if (key.startsWith(ConfigChangeWatcher.WatchType.GROUP.name())) {
```

Comment:
What about adding some delimiter like `GROUP::` or `GROUP.` or `GROUP.`?

Reply:
I think we have the enough metadata in the memory, why do we need to check from name's prefix? @wankai123

Reply:
yes, It can get from `WatcherHolder` by changing the class authority. we don't need to check from the name's prefix.
and then should we keep the name's prefix `GROUP` to identify it's a `GROUP key`? @wu-sheng wu

Reply:
I think we don't need to add prefix. The one provides the feature based on this, they should show this clearly on the doc. We could add a new column on the table(dynamic configuration table), which should be more clear.